creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -116,6 +117,23 @@ const handleSyncFromPrimary = async ({
         }
       }
 
+      // Store this user's file paths in redis to delete later (after wiping db)
+      let numFilesToDelete = 0
+      try {
+        numFilesToDelete = await DiskManager.gatherCNodeUserDataToDelete(
+          wallet,
+          logger
+        )
+      } catch (error) {
+        await DiskManager.clearFilePathsToDelete(wallet)
+        errorResponse = {
+          error,
+          result: 'failure_delete_disk_data'
+        }
+
+        throw error
+      }
```

Comment:
interesting that we error sync on failure to gatherCNodeUserDataToDelete but don't error sync on failure to actually delete those files

Reply:
> interesting that we error sync on failure to gatherCNodeUserDataToDelete but don't error sync on failure to actually delete those files

we do error the sync on an unexpected error with the larger file deletion function, but yeah we skip `fs.unlink()` errors since some are bound to happen. I added a log `"Could not delete file: {reason}"` so we can debug if something is being deleted for some crazy unexpected reason

example of error that's bound to happen: redis set has a storage path for a parent dir of images and storage paths of its child images. the first path fetched from redis is the parent dir, so we recursively delete that. next path is the child, but now that file doesn't exist and throws an error we don't care about -- I've seen this happen locally.

Reply:
right, makes sense

Reply:
> does that result not need to be a prom constant? or is that diff

yep great catch. just added the `failure_delete_disk_data` value option for the `result` label now